understanding/redundant-entry.html

```diff
@@ -56,191 +55,142 @@
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>簡潔に</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Make it easier for users to complete multi-step processes.</dd>
+                  <dt>目標</dt>
+                  <dd>ユーザーが複数ステップのプロセスを容易に完了できるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Don't ask for the same information twice in the same session.</dd>
+                  <dt>何をするか</dt>
+                  <dd>同じセッションで同じ情報を 2 回要求しない。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Some people with cognitive disabilities have difficulty remembering what they entered
-                     before.
+                  <dt>なぜ重要か</dt>
+                  <dd>認知障害のある人の中には、以前に入力したものを思い出すのが困難な人がいる。
                   </dd>
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to ensure that users can successfully complete
-                  multi-step processes. It reduces cognitive effort where information is asked for more
-                  than once during a process. It also reduces the need to recall information provided
-                  in a previous step.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、ユーザーが複数ステップのプロセスを確実に完了できるようにすることである。これにより、プロセス中に情報が複数回要求される場合の認知的労力が軽減される。また、前のステップで提供した情報を思い出す必要性も軽減される。
                </p>
-               <p>Information that is required to be remembered for input can pose a significant barrier
-                  to users with cognitive or memory difficulties. All users experience a natural gradual
-                  mental fatigue as they proceed through steps in a process. This fatigue is accelerated
-                  by the stress of recalling information from short-term working memory. Users with
-                  learning, and cognitive disabilities are highly susceptible to mental fatigue.
+               <p>入力のために記憶する必要がある情報は、認知又は記憶に問題のある利用者にとって大きな障壁となる可能性がある。すべての利用者は、プロセスのステップを進めるにつれて、自然と段階的に精神的な疲労を感じる。この疲労は、短期の作業記憶から情報を思い出すストレスによって加速される。学習障害及び認知障害のある利用者は、精神的な疲労の影響を非常に受けやすい。
                </p>
-               <p>Requiring people to recall information previously entered can cause them to give up
-                  or re-enter the same information incorrectly. The autocomplete feature of browsers
-                  is not considered sufficient because it is the content (the web site) that needs to
-                  provide the stored information for a redundant entry, or avoid asking for the same
-                  information again.
+               <p>以前に入力した情報を思い出すように要求すると、同じ情報を諦める、又は間違った再入力する可能性がある。コンテンツ (ウェブサイト) が、冗長なエントリーのために保存された情報を提供する、又は同じ情報を再度要求しないようにする必要があるため、ブラウザーのオートコンプリート機能は十分とはみなされない。
```

Comment:
「同じ情報を諦める」という表現に若干の違和感がある気がしました。また、「冗長なエントリー」は達成基準名称に合わせて「冗長な入力項目」でしょうか。

```suggestion
               <p>以前に入力した情報を思い出すように要求すると、利用者は諦める、又は同じ情報を間違って再入力する可能性がある。コンテンツ (ウェブサイト) が、冗長な入力項目のために保存された情報を提供する、又は同じ情報を再度要求しないようにする必要があるため、ブラウザーのオートコンプリート機能は十分とはみなされない。
```

Reply:
@caztcha 

ありがとうございます。ここではuserではなくpeopleなので人々としてみましたがどうでしょう
（よく見るとcause A to doなので、人々に～させるなのですが、そのまま取るとあまりよろしくないですね…）

```suggestion
               <p>以前に入力した情報を思い出すように要求すると、人々は諦める、又は同じ情報を間違って再入力する可能性がある。コンテンツ (ウェブサイト) が、冗長な入力項目のために保存された情報を提供する、又は同じ情報を再度要求しないようにする必要があるため、ブラウザーのオートコンプリート機能は十分とはみなされない。
```

Reply:
たしかに、people ですね。ご指摘ありがとうございます！ご提案にて commit します。